CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ project(seam-carving)
 
 # var setup
 set(INCLUDE_DIRS include/)
-set(SRC_FILES src/energy.cpp src/CarvableImage.cpp)
```

Comment:
`CarvableImage.cpp` is removed here because it will be removed in the next major release. No need to have it lingering around now.